science/gds/Portfile

```diff
@@ -70,10 +66,28 @@ destroot.args-append \
               pkgpythondir="${python.pkgd}/${name}" \
               pkgpyexecdir="${python.pkgd}/${name}"
 
-#
+#------------------------------------------------------------------------
```

Comment:
Please remove extraneous comments, whose only purpose is to highlight another comment line, like the above one.